apps/nextjs-website/src/lib/api.ts

```diff
@@ -145,6 +145,37 @@ export async function getVisibleInHomeWebinars(): Promise<readonly Webinar[]> {
   return webinars.filter((webinar) => webinar.isVisibleInHome);
 }
 
+export async function getVisibleInHomeFutureWebinars(): Promise<
+  readonly Webinar[]
+> {
+  return webinars.filter((webinar) => {
+    const startDateTimestamp =
+      webinar?.startDateTime && new Date(webinar.startDateTime).getTime();
+
+    return (
+      webinar.isVisibleInHome &&
+      (!startDateTimestamp || startDateTimestamp > new Date().getTime())
+    );
+  });
```

Comment:
You should also order them as we did in `apps/nextjs-website/src/components/organisms/WebinarsTemplate/WebinarsTemplate.tsx`.
It would help if you also considered using the same function by doing a little refactoring of `WebinarsTemplate`